osgp/platform/osgp-adapter-domain-publiclighting/src/main/java/org/opensmartgridplatform/adapter/domain/publiclighting/application/tasks/DeviceConnection104LmdScheduledTask.java

```diff
@@ -40,12 +40,12 @@ public void run() {
         LOGGER.info("Ensuring active connections with IEC 60870-5-104 light measurement devices");
         try {
             final List<LightMeasurementDevice> devices = this.findLightMeasurementDevicesByProtocol(PROTOCOL);
-            final List<LightMeasurementDevice> devicesToConnect = this.findLightMeasurementDevicesToConnect(devices,
+            final List<LightMeasurementDevice> deviceToConnect = this.findLightMeasurementDevicesToConnect(devices,
```

Comment:
Why was this renamed to `deviceToConnect`? The variable holds a list of devices so plural (`devicesToConnect`) seems to make more sense.